src/phaser-game/utils/playerMove.ts

```diff
@@ -0,0 +1,53 @@
+import { PLAYER_MOVEMENT } from './constants';
+
+const movePlayer = (player: any, cursors: any, playerText: any) => {
+  let playerMoved = false;
+  let speed = 1;
+  if (cursors.shift.isDown) {
+    speed = 1.5;
+  }
+  if (cursors.up.isDown) {
```

Comment:
Just out a comment but maybe it would be beneficial to use case statements next time or even better jump table methodology